spack/package.py

```diff
@@ -69,14 +69,14 @@ class Arbor(CMakePackage, CudaPackage):
 
     # python (bindings)
     extends("python", when="+python")
-    depends_on("python@3.7:", when="+python", type=("build", "run"))
+    depends_on("python@3.6:", when="+python", type=("build", "run"))
     depends_on("py-numpy", when="+python", type=("build", "run"))
     with when("+python"):
-        depends_on("py-pybind11@2.6:", type=("build", "run"))
-        depends_on("py-pybind11@2.8.1:", when="@0.5.3:", type=("build", "run"))
+        depends_on("py-pybind11@2.6:", type=("build"))
+        depends_on("py-pybind11@2.8.1:", when="@0.5.3:", type=("build"))
```

Comment:
Here's another question:
If we did not update this line **only** 0.5.3 would get 2.8.1? Meaning shouldn't it be
>=0.5.3 or Spack's equivalent? Or can we rely on CI to catch this?

Reply:
The trailing colon signifies 'and up'.

Reply:
That's a sneaky bit of syntax